bot/package.json

```diff
@@ -1,34 +1,37 @@
 {
   "name": "airhornbot",
   "version": "1.0.0",
+  "description": "",
   "main": "index.js",
-  "author": "Discord",
+  "author": "Sky W",
   "license": "MIT",
-  "private": true,
```

Comment:
Nah you wanna keep that to prevent accidental publishing of the site to the npm registry